sentry_sdk/profiler.py

```diff
@@ -308,13 +309,11 @@ def slice_profile(self, start_ns, stop_ns):
                         )
                     current_stack.append(frames[frame])
 
-                try:
-                    stack_idx = stacks.index(current_stack)
-                except ValueError:
-                    stack_idx = len(stacks)
-                    stacks.append(current_stack)
+                if current_stack not in stacks:
+                    stacks[current_stack] = len(stacks)
+                    stacks_list.append(current_stack)
```

Comment:
What's this for? You just append to it and never use it.

Reply:
I should have returned it instead of `stacks`.